ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-stable-4.15-upgrade-from-stable-4.15.yaml

```diff
@@ -78,18 +78,16 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-upgrade-qe-sanity
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
-- as: azure-ipi-des-mini-perm-amd-mixarch-f28
+- as: azure-ipi-des-mini-perm-arm-f28
```

Comment:
It seems there is no same job in `openshift-openshift-tests-private-release-4.15__multi-stable.yaml`, maybe we can remove it?

Reply:
Done.